src/booleans.js

```diff
@@ -1,61 +1,67 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b;
 };
 
 function none(a, b) {
-  // your code here
+  return !(a || b);
 };
 
 function one(a, b) {
-  // your code here
+  return a !== b;
 };
 
 function truthiness(a) {
-  // your code here
+  return !!(a);
 };
 
 function isEqual(a, b) {
-  // your code here
+  return a === b;
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  return a > b;
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  return a <= b;
 };
 
 function isOdd(a) {
-  // your code here
+  return a % 2 !== 0;
 };
 
 function isEven(a) {
-  // your code here
+  return a % 2 == 0;
 };
 
 function isSquare(a) {
-  // your code here
+  return Math.sqrt(a) % 1 === 0;
 };
 
 function startsWith(char, string) {
-  // your code here
+  if (char === string[0]) {
+    return true;
+  }
+  return false;
 };
 
 function containsVowels(string) {
-  // your code here
+  if (string.match(/[a|e|i|o|u|]/i, 0)) {
+    return true;
+  }
+  return false;
```

Comment:
As above, you can simplify this a bit. The `.match` function in the way youre using it here will either return the first instance it finds, or `null` if it finds no matches, so we can turn this into a boolean using the double negation trick (!!) that you are using for the truthiness function above:
```
  return !!string.match(/[a|e|i|o|u|]/i, 0);
```
Here if the result is a match, we dont want to return the value but a `true` boolean, so we can use a single ! to turn that truthy value false, then another to flip it back to true as we want.
If there is no match it will return `null` (a falsy value but _not strictly equal to false_), by the same logic we can flip it to truthy and back to falsy and give us the result we want :)